tools/serverpod_cli/lib/src/analyzer/dart/endpoint_analyzers/endpoint_method_analyzer.dart

```diff
@@ -124,7 +124,14 @@ abstract class EndpointMethodAnalyzer {
 
     var innerType = typeArguments[0];
 
-    if (innerType is VoidType) {
+    if (innerType is VoidType && dartType.isDartAsyncStream) {
+      return SourceSpanSeverityException(
+        'The type void is not supported for streams.',
```

Comment:
Suggestion(NIT): 
```suggestion
        'The type "void" is not supported for streams.',
```

This makes it the same as the description on line 149 and makes it a bit clearer what the type is.